internal/pkg/slicer/slicer.go

```diff
@@ -98,11 +98,6 @@ func SliceTable(logger log.Logger, table Table) (err error) {
 	if err != nil {
 		return err
 	}
-	defer func() {
-		if closeErr := writer.Close(); closeErr != nil && err == nil {
-			err = closeErr
-		}
-	}()
```

Comment:
Writer sa zatvaral az uplne nakoniec, po zalogovani velkosti.
Teda zaver suboru este nemusel byt zapisany na disku.

`defer` je nieco ako `finally`, vykonava sa to na konci funkcie v LIFO poradi.